write path: handle created timestamp (#11977)
grafana/mimir: CHANGELOG.md

  * `memberlist.acquire-writer-timeout`: `1s`
    These defaults perform better but may cause long-running packets to be dropped in high-latency networks.
* [CHANGE] Query-frontend: Apply query pruning and check for disabled experimental functions earlier in query processing. #11939

> * [FEATURE] Distributor: Experimental support for Prometheus Remote-Write 2.0 protocol. Limitations: if samples in a write request are not ordered by time, the created timestamp might be dropped, per series metadata is merged on metric family level automatically, ingestion might fail if client sends ProtoBuf fields out of order. The label `version` is added to the metric `cortex_distributor_requests_in_total` with a value of either `1.0` or `2.0` depending on the detected Remote-Write protocol. #11100 #11101 #11192 #11143 #11977

Reviewer: I tried to break this up a bit for clarity. Let me know if the meaning is preserved.

> * [FEATURE] Ingester/Block-builder: handle the Created timestamp field of write requests. #11977

Reviewer: ```suggestion
* [FEATURE] Ingester/Block-builder: Handle the timestamp field for write requests. #11977
```
